### ISSUE TITLE
Pa11y Not Running in GH Actions

### ISSUE THREAD

**Issue by btylerburton** [GSA/data.gov#4273]
Datagov would like to run `pa11y-ci` on the pages in our repo, but `pa11y-ci` is failing silently.

## How to reproduce

1. Run tests in https://github.com/GSA/datagov-11ty repo.

## Expected behavior

I expect to see a report of all the URLs and their issues, see example below.

## Actual behavior

The pa11y report fails silently.

## Sketch

- [ ] Build off [`pa11y-debug` branch](https://github.com/GSA/datagov-11ty/tree/pa11y-debug). Contains updated NPM script and patch to `pa11y-ci` needed to expose the runtime error.
- [ ] Explore the options related to [running pa11y-ci in a Docker container](https://github.com/pa11y/pa11y-ci/blob/e7b7c17b4ec5fa5d3b52b539f15b520af470c0b2/README.md#docker)

**Comment by nickumia-reisys** [comment]
Is this the failing silently message?
```bash
Running Pa11y on 153 URLs:
Running Pa11y on URL https://data.gov/privacy-policy/
Using a pre-configured Headless Chrome instance, the `chromeLaunchConfig` option will be ignored
 > https://data.gov/privacy-policy/ - Failed to run >> TimeoutError: Pa11y timed out (60000ms)
```

**Comment by btylerburton** [comment]
@nickumia-reisys yes. without the patch all we were getting was `Failed to run`. My guess is it's a missing puppetteer dependency.

**Comment by hkdctol** [comment]
Moving to icebox for the time being.
